salt/metalk8s/kubeadm/init/kubelet-start/configured.sls

```diff
@@ -0,0 +1,62 @@
+{%- from "metalk8s/map.jinja" import kubelet with context %}
+
+Create kubelet service environment file:
+  file.managed:
+    - name: {{ kubelet.service.environment_file }}
+    - source: salt://metalk8s/kubeadm/init/kubelet-start/files/kubeadm.env
+    - template: jinja
+    - user: root
+    - group: root
+    - mode: 644
+    - context:
+        options: {{ kubelet.service.options }}
+
+{#- Default dataset for kubelet config #}
+{%- set dataset = salt['grains.filter_by']({
+  'default': {
+    'kind': 'KubeletConfiguration',
+    'authentication': {
+      'anonymous': {'enabled': true},
+      'webhook': {'enabled': false}
+    },
+    'authorization': {'mode': 'AlwaysAllow'}
+  }
+}, merge=kubelet.config) %}
+Create kubelet config file:
```

Comment:
This applies to other files created as well (like the environment file above), but: https://github.com/scality/metalk8s/blob/development/2.0/docs/requirements.rst#tagged-generated-files

Reply:
Oh yes True i will do it

Reply:
Btw for a `file.serialize` i don't find any "real" way to add a comment

Reply:
Ah, too bad. Maybe we should do a custom/derived module one day. OK then, for now, though maybe we should create a tech-debt ticket to track cases like this.

Reply:
Or if it's only about dump a file then we can use a file.managed and just dump a yaml file. But if one day we just want to merge them then .... I don't know